Resources/Prototypes/Entities/Objects/Consumable/Food/skewer.yml

```diff
@@ -50,7 +50,7 @@
 # Custom Kebab Example
 
 - type: entity
-  name: Hawaiian kebab
+  name: hawaiian kebab
```

Comment:
Hawaii is a place, and therefore a proper noun. Proper nouns should remain capitalized.